docs/custom_build.md

```diff
@@ -20,9 +34,56 @@ custom_build(
 )
 ```
 
-One piece of this needs more explanation: `$EXPECTED_REF`.
+Tilt will run this command to build the image, verify that the image is in Docker,
+then push the image to the appropriate image registry.
+
+You can also use this to use `docker` flags that the `docker_build()` function doesn't support.
+
+### Buildah (or any Image Builder Independent of Docker)
+
+[Buildah](https://buildah.io/) is an independent Docker image builder.
+
+Buildah has its own API and own image store. A `custom_build()` call needs to
+both build and push the image.
+
+```python
+custom_build(
+  'frontend',
+  'buildah bud -t $EXPECTED_REF frontend && buildah push $EXPECTED_REF $EXPECTED_REF',
+  ['./frontend'],
+  skips_local_docker=True,
+)
+```
+
+The `skips_local_docker` parameter tells Tilt that we don't expect the image to
+ever show up in the local Docker image store, so it won't be able to verify the
+image locally.
+
+There are a couple of caveats you should be aware of with `buildah` (and similar builders):
+
+- They often require privileged access. You may need to run tilt with `sudo` or
+  inside an appropriate sandbox
+
+- If you're using Tilt to push to an insecure registry, you will need to
+  configure your builder for that registry. For example, to use `buildah` with
+  Microk8s, you need to add `localhost:32000` to
+  [registries.insecure](https://github.com/containers/buildah/blob/master/install.md#registriesconf)
+
+### Bazel (or any Image Builder that Interoperates with Docker)
```

Comment:
see above about title casing

Reply:
but also, this seems like a simpler case than `buildah` et al, I might put this section above "image builders independent of docker" so the doc progresses from most dockery --> least.

Reply:
done